include/up-cpp/datamodel/validator/UUri.h

```diff
@@ -73,12 +89,17 @@ using ValidationResult = std::tuple<bool, std::optional<Reason>>;
 /// resource_id must be 0.
 [[nodiscard]] ValidationResult isValidRpcResponse(const v1::UUri&);
 
-/// @brief Checks if UUri is valid as a default source on a UTransport.
+/// @brief Checks if UUri is valid as an entity URI for a UTransport.
 ///
-/// @note This just calls isValidRpcResponse() because the requirements are the
-///       same in both cases. It is provided as a separate function here for
-///       improved readability where it is called.
-[[nodiscard]] ValidationResult isValidDefaultSource(const v1::UUri&);
+/// @note The requirements for this URI are identical to isValidRpcResponse()
+///       except that the authority name is not allowed to be blank.
+[[nodiscard]] ValidationResult isValidDefaultEntity(const v1::UUri&);
+
+/// @note DEPRECATED
+/// @see isValidDefaultEntity()
+/// @brief Checks if UUri is valid as a default source on a UTransport.
+[[deprecated]] [[nodiscard]] ValidationResult isValidDefaultSource(
```

Comment:
In the deprecated isValidDefaultSource, can we also get "deprecated" to add a string to say "Use isValidDefaultEntity instead" ??